### PR TITLE
adding Arabic md files and fix numbering issues in Arabic PDF

### PR DESCRIPTION
**Pull request by aref2008**
This Pull Request relates to issue #...

For context: RTL needed 
For tables in Microsoft word: table properties -> Table direction -> Right to Left
For Numbers: reversed to match Arabic (no need to work with it)

### CODE REVIEW

File: 4.0/ar/0x13-V5-Validation-Sanitization-Encoding.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# V5 ت5: التحقق من الصحة Validation والتعقيم Sanitization  والترميز Encoding
```
Comment (tghosth) [inline 4.0/ar/0x13-V5-Validation-Sanitization-Encoding.md:1]: @aref2008 
Should this be:
# ت5: التحقق من الصحة Validation والتعقيم Sanitization  والترميز Encoding


?

Comment (bangot-zz) [reply 4.0/ar/0x13-V5-Validation-Sanitization-Encoding.md:1]: Hi Josh

That's right!
They forgot to remove "V5". the equivalent Arabic word of it is "ت5"

could you modify it as you suggest.

regards

**Comment by tghosth** [comment]
Hi @jmanico you deleted a comment here, do you know why?

**Review by tghosth — APPROVED** [review]

**Comment by tghosth** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v4.0.3|[](https://github.com/OWASP/ASVS/pull/1214)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by tghosth** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v4.0.3|[](https://github.com/OWASP/ASVS/pull/1214)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by tghosth** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v4.0.3|[](https://github.com/OWASP/ASVS/pull/1214)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by tghosth** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v4.0.3|[](https://github.com/OWASP/ASVS/pull/1214)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**